plugins/device-injector/README.md

```diff
@@ -101,11 +101,28 @@ The annotation value syntax for mount injection is
     ...
 ```
 
+## Deployment
+
+The NRI repository contains kustomize overlays for this plugin at
+[contrib/kustomize/device-injector](../../contrib/kustomize/device-injector).
+
+Deploy the latest release with:
+
+```bash
+kubectl apply -k https://github.com/containerd/nri/contrib/kustomize/device-injector
+```
+
+Deploy the latest development build from tip of the main branch with:
+
+```bash
+kubectl apply -k https://github.com/containerd/nri/contrib/kustomize/device-injector/unstable
+```
+
```

Comment:
@marquiz: I think it could be worth mentioning the specific version installing trick, because I suspect it might not be obvious for the average user.

```
kubectl create -k "https://github.com/containerd/nri/contrib/kustomize/hook-injector?ref=v0.10.0"
```